### PR TITLE
CDPT-1549 Update domain and terminology (docs -> howto)

### PR DESCRIPTION
**Pull request by EarthlingDavey**

### CODE REVIEW

File: .github/pages/index.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
 ---
-title:  "Editor Documentation"
+title:  "HowTo for Admins"
```
Comment (wilson1000) [inline .github/pages/index.md:2]: Could we rename this to **HowTo for Content Editors** - in our context _admin_ would (more often than not) refer to `wp-admin` or, _the Dashboard_.

File: .github/pages/index.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
 nav_exclude: true
 ---
 
-# Editor Documentation
-
 {: .fs-6 }
-Documentation for editors of the Justice UK website. Guides to creating, editing and scheduling content.
+Welcome to the HowTo for Justice UK Admins. Guides to creating, editing and scheduling content.
```
Comment (wilson1000) [inline .github/pages/index.md:7]: The reference to Admin could read: 
- Welcome to the HowTo for the Justice UK Dashboard, or
- Welcome to the HowTo for Justice UK admin pages

Something like that 😄

**Review by wilson1000 — CHANGES_REQUESTED** [review]
Just a couple of comments @EarthlingDavey - great work mate!

File: .github/pages/index.md (hunk 2 above)
Comment (wilson1000) [reply .github/pages/index.md:7]: Or something like:
- Welcome to the HowTo Guide for Justice UK admin pages

Maybe?

Comment (EarthlingDavey) [reply .github/pages/index.md:7]: Cheers! I'll go with: 

> Welcome to the HowTo Guide for Justice UK admin pages

**Pull request merged**